sample-java-model/src/main/java/com/vimeo/sample_java_model/SwappableParserExampleModel.java

```diff
@@ -0,0 +1,28 @@
+package com.vimeo.sample_java_model;
+
+import com.vimeo.stag.UseStag;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * A model to test the case where the consumer
+ * might try to use different parsers for
+ * an object across different gson instances.
+ */
+@UseStag
+public class SwappableParserExampleModel {
```

Comment:
It seems this file is only used for unit tests. Should it be in the test package? Should this package `private` rather `public`?